### PR TITLE
Update disabling-localised-urls.rst

### PR DESCRIPTION
**Pull request by hmonglee**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: docs/cookbook/shop/disabling-localised-urls.rst [hunk 1]
```diff
@@ -31,38 +34,54 @@ With:
...
+        resource: "@SyliusShopBundle/Resources/config/routing/payum.yml"
+
+**2.** Customise the security settings in the ``app/config/packages/security.yml``.
 
 Replace:
 
 .. code-block:: yaml
 
-    # app/config/security.yml
+    # app/config/packages/security.yml
```
Comment (CoderMaggie) [inline docs/cookbook/shop/disabling-localised-urls.rst]: there is no `app/` prefix now :)

File: docs/cookbook/shop/disabling-localised-urls.rst [hunk 2]
```diff
@@ -31,38 +34,54 @@ With:
...
 
     parameters:
         # ...
         sylius.security.shop_regex: "^"
 
-**3.** Customise SyliusShopBundle to use storage-based locale switching by adding the following lines at the end of the ``app/config/config.yml``.
+**3.** Customise SyliusShopBundle to use storage-based locale switching in the ``app/config/packages/_sylius.yml``.
```
Comment (CoderMaggie) [inline docs/cookbook/shop/disabling-localised-urls.rst]: <del>app/</del>

File: docs/cookbook/shop/disabling-localised-urls.rst [hunk 3]
```diff
@@ -31,38 +34,54 @@ With:
...
         # ...
-        sylius.security.shop_regex: "^/(?!admin|api)[^/]++"
+        sylius.security.shop_regex: "^/(?!admin|api/.*|api$|media/.*)[^/]++""
 
 With:
 
 .. code-block:: yaml
 
-    # app/config/security.yml
+    # app/config/packages/security.yml
```
Comment (CoderMaggie) [inline docs/cookbook/shop/disabling-localised-urls.rst]: <del>app/</del>

File: docs/cookbook/shop/disabling-localised-urls.rst [hunk 4]
```diff
@@ -31,38 +34,54 @@ With:
...
+    sylius_shop:
+        product_grid:
+            include_all_descendants: true
+
+With:
 
 .. code-block:: yaml
 
-    # app/config/config.yml
+    # app/config/packages/_sylius.yml
```
Comment (CoderMaggie) [inline docs/cookbook/shop/disabling-localised-urls.rst]: <del>app/</del>

File: docs/cookbook/shop/disabling-localised-urls.rst [hunk 5]
```diff
@@ -31,38 +34,54 @@ With:
...
-
+    # config/routes/sylius_shop.yaml    
     sylius_shop:
         resource: "@SyliusShopBundle/Resources/config/routing.yml"
 
-**2.** Customise the security settings in the ``app/config/security.yml``.
+    sylius_shop_payum:
+        resource: "@SyliusShopBundle/Resources/config/routing/payum.yml"
+
+**2.** Customise the security settings in the ``app/config/packages/security.yml``.
```
Comment (CoderMaggie) [inline docs/cookbook/shop/disabling-localised-urls.rst]: <del>app/</del>

**Comment by hmonglee** [comment]
@CoderMaggie thx, you're right, done

**Comment by pamil** [comment]
Thank you, @hmonglee! 🎉

**Pull request merged**